pkg/instrumentation/categories.go

```diff
@@ -79,7 +79,24 @@ func DetermineCategoryFromArgs(args []string, knownCommands []string, flagsAllow
 	return result
 }
 
-var KNOWN_COMMANDS = []string{"test", "code", "monitor", "iac", "rules", "describe", "container", "debug", "config"}
+var KNOWN_COMMANDS = []string{
+	"test",
+	"code",
+	"monitor",
+	"iac",
+	"rules",
+	"describe",
+	"container",
+	"debug",
+	"config",
+	"auth",
+	"ignore",
+	"log4shell",
+	"policy",
+	"fix",
+	"apps",
```

Comment:
Question: any reason the commands below are missing?
* iac capture
* iac rules init
* iac rules push
* iac update-exclude-policy

Reply:
The logic for building the category array parses the command line one argument at a time, it doesn't check "iac capture" as a single command. 

I'm assuming that this slice is used for that purpose only. If we need this, we also need to tweak the DetermineCategoryFromArgs method. 

Does this make sense?

Reply:
I understand that the list of known_commands contains just the individual command names not set in relation. That makes sense. 
My question is more about why the names "capture" or "init" are not there but the "rules" or "describe". Everything in this list is supposed to be known commands, so why not add all subcommands?

Reply:
Ah! ..you're right, nvm. I pushed the missing commands.

Reply:
Perfect, thanks!